src/Components/CreatorsStoreStyled.js

```diff
@@ -0,0 +1,262 @@
+  margin: 0 auto;
+`;
+
+export const CampaignMarketParticlesWrapper = styled.div`
+  position: absolute;
+  top: 18px;
+  right: 0;
+
+  img {
+    max-width: 28px;
+    width: 100%;
+    height: auto;
+    max-height: 38px;
+    box-sizing: 'border-box';
+  }
+
+  @media (max-width: 766px) {
+    display: none;
+  }
+`;
+
+export const CreatorStyledInnerWrapper = styled.div`
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  @media (max-width: 766px) {
+    flex-direction: column;
+    align-items: flex-start;
+  }
+`;
+
+export const RightInnerWrapper = styled.div`
+  margin: 0 0 0 30px;
+  max-width: 48%;
+  width: 100%;
+  display: flex;
+  font-family: 'ProximaNova', sans-serif;
+
+  @media (max-width: 766px) {
+    max-width: 100%;
+    margin: -20px 0 0 10px;
+  }
+`;
+
+export const CreatorStyledLeftInner = styled.div`
+  max-width: 48%;
+  width: 100%;
+  position: relative;
+  font-family: 'ProximaNova', sans-serif;
+`;
+
+export const CreatorStyledRateBox = styled.div`
+  position: absolute;
+  right: 112px;
+  top: -13px;
+  padding: 10px;
+  font-family: 'ProximaNova', sans-serif;
+  background: #ffffff;
+  box-shadow: rgba(240, 97, 86, 0.1) 5px 12px 21px;
+
+  @media (max-width: 991px) {
+    right: 120px;
+    top: 23px;
+    padding: 6px 6px;
+  }
+
+  @media (max-width: 720px) {
+    right: -4px;
+    top: 23px;
+    padding: 6px 6px;
+  }
+`;
+
+export const StyledRateBoxHeading = styled.h5`
+  font-size: 14px;
+  font-family: 'ProximaNova', sans-serif;
+  line-height: 14px;
+  font-weight: bold;
+  color: #ff4e53;
+  margin: 2px;
+  font-weight: bold;
+  @media (max-width: 991px) {
+    font-size: 12px;
+  }
+  @media (max-width: 767px) {
+    font-size: 12px;
+  }
+`;
+
+export const StyledRateParagraph = styled.p`
+  font-family: 'ProximaNova', sans-serif;
+  margin: 0;
+  font-size: 14px;
+  line-height: 22px;
+
+  @media (max-width: 991px) {
+    font-size: 10px;
+  }
+
+  @media (max-width: 767px) {
+    font-size: 10px;
+  }
+`;
+
+export const ImageWrapper = styled.div`
+  img {
+    max-width: 398px;
+    width: 100%;
+    top: -220px;
+  }
+
+  @media (max-width: 991px) {
+    margin: 40px 0 0 10px;
+    img {
+      max-width: 298px;
+      width: 100%;
+    }
+  }
+`;
+
+export const CreatorsStore = styled.div`
+  font-size: 15px;
+  line-height: 1.5;
+  top: 20px;
+  right: 20px;
+  font-weight: 400;
+  color: black;
+  background: linear-gradient(50deg, white, transparent);
+  padding: 8px 12px;
+  margin: 0 0 25px 20px;
+  width: max-content;
+  opacity: 1;
+  transform: translate3d(0, 0, 0) translateX(0);
+
+  @media (max-width: 766px) {
+    font-size: 11px;
+    margin: 0 0 0 -20px;
+  }
+`;
+
+export const PopupWrapper = styled.div`
+  &::after {
+    position: absolute;
+    content: '';
+    background: linear-gradient(240deg, #ff4e53 70%, #f06156 100%);
+    left: -650px;
+    transform: translateX(-55%);
+    padding: 20px;
+    opacity: 0.8;
+    width: 2550px;
```

Comment:
why 2550?? @aeshapatel025

Reply:
It is a part of that pink line of spread page that`s why i have given 2550px width to it